pkg/cacheutil/memcached_client.go

```diff
@@ -54,6 +57,11 @@ var (
 		MaxGetMultiBatchSize:      0,
 		DNSProviderUpdateInterval: 10 * time.Second,
 		AutoDiscovery:             false,
+		SetAsyncCircuitBreakerHalfOpenMaxRequests: 10,
+		SetAsyncCircuitBreakerOpenDuration:        5 * time.Second,
+		SetAsyncCircuitBreakerMinRequests:         50,
+		SetAsyncCircuitBreakerConsecutiveFailures: 5,
+		SetAsyncCircuitBreakerFailurePercent:      0.05,
```

Comment:
We might need to keep an eye on this to make sure those values sane.
So far, it looks good to me. Would be better to have another pair of eyes on it